x/avs/types/expected_keepers.go

```diff
@@ -22,3 +29,28 @@ type BankKeeper interface {
 	SpendableCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
 	// Methods imported from bank should be defined here
 }
+
+// EVMKeeper defines the expected EVM keeper interface used on erc20
+type EVMKeeper interface {
+	SetAccount(ctx sdk.Context, addr common.Address, account statedb.Account) error
```

Comment:
If you only use the `SetCode` function, you can remove the other functions from this interface.